### PR TITLE
add paragraph of text

### PR DESCRIPTION
**Pull request by ish-codes**
- one paragraph of hipster ipsum added

### CODE REVIEW

**Review by code-mich — CHANGES_REQUESTED** [review]
looking great pal

File: index.html [hunk 1]
```diff
@@ -10,6 +10,15 @@
 
 <body>
 	<h1>Wowee woo sick page!</h1>
+	<p>+1 tumeric direct trade live-edge, jean shorts drinking vinegar four dollar toast DIY celiac artisan portland.
```
[the remark this answers is not in the document]
Comment (ish-codes) [reply index.html:13]: i've removed the last sentence 🗑️

**Comment by ish-codes** [comment]
made requested changes.

**Review by code-mich — APPROVED** [review]
looks great, go ahead and merge

**Pull request merged**